master/internal/db/postgres_checkpoints.go

```diff
@@ -155,17 +155,26 @@ func UpdateCheckpointSizeTx(ctx context.Context, idb bun.IDB, checkpoints []uuid
 	var experimentIDs []int
 	err := idb.NewRaw(`
 UPDATE trials SET checkpoint_size=sub.size, checkpoint_count=sub.count FROM (
-	SELECT trial_id,
-	COALESCE(SUM(size) FILTER (WHERE state != 'DELETED'), 0) AS size,
-	COUNT(*) FILTER (WHERE state != 'DELETED') AS count
-	FROM checkpoints_view
-	WHERE trial_id IN (
+	SELECT t.id AS trial_id,
+	COALESCE(SUM(size) FILTER (WHERE c.state != 'DELETED'), 0) AS size,
+	COUNT(*) FILTER (WHERE c.state != 'DELETED') AS count
+	FROM raw_checkpoints c INNER JOIN trials t on c.trial_id = t.id
+	WHERE t.id IN (
 		SELECT trial_id FROM checkpoints_view WHERE uuid IN (?)
 	)
-	GROUP BY trial_id
+	GROUP BY t.id
+	UNION
+	SELECT t.id AS trial_id,
+	COALESCE(SUM(size) FILTER (WHERE checkpoints_v2.state != 'DELETED'), 0) AS size,
+	COUNT(*) FILTER (WHERE checkpoints_v2.state != 'DELETED') AS count
+	FROM checkpoints_v2 INNER JOIN trials t on checkpoints_v2.task_id = t.task_id
+	WHERE t.id IN (
+		SELECT trial_id FROM checkpoints_view WHERE uuid IN (?)
```

Comment:
you know what, we should be able to get rid of `checkpoints_view` here altogether. give me a minute.

Reply:
https://github.com/determined-ai/determined/pull/6390/commits/cb783b24f2ef7cf1813bc861e8afca18673b5bfb